Integrations/git/git.py

```diff
@@ -0,0 +1,306 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+import json
+import requests
+from datetime import datetime, timedelta
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+USER = demisto.params().get('user')
+TOKEN = demisto.params().get('token')
+BASE_URL = demisto.params().get('url')
+REP = demisto.params().get('repository')
+USE_SSL = not demisto.params().get('insecure', False)
+FETCH_TIME = demisto.params().get('fetch_time', '30 days')
+
+USER_SUFFIX = '/repos/{}/{}'.format(USER, REP)
+ISSUE_SUFFIX = '/issues'
+RELEASE_SUFFIX = '/releases'
+
+# Headers to be sent in requests
+HEADERS = {
+    'Authorization': "Bearer " + TOKEN
+}
+# Remove proxy if not set to true in params
+if not demisto.params().get('proxy'):
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
+
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, url_suffix, params=None, data=None):
+    # A wrapper for requests lib to send our requests and handle requests and responses better
+    res = requests.request(
+        method,
+        BASE_URL + url_suffix,
+        verify=USE_SSL,
+        params=params,
+        data=json.dumps(data),
+        headers=HEADERS
+    )
+    # Handle error responses gracefully
+    if res.status_code >= 400:
+        return_error('Error in API call to Example Integration [%d] - %s' % (res.status_code, res.reason))
+
+    return res.json()
+
+
+def data_formatting(title, body, milestone, labels, assignees, state):
+    """This method creates a dictionary to be used as "data" field in an http request."""
+    data = {}
+    if title is not None:
+        data['title'] = title
+
+    if body is not None:
+        data['body'] = body
+
+    if state is not None:
+        data['state'] = state
+
+    if milestone is not None:
+        data['milestone'] = int(milestone)
+
+    if labels is not None:
+        data['labels'] = labels.split(',')
+
+    if assignees is not None:
+        data['assignees'] = assignees.split(',')
+
+    return data
+
+
+def context_create_issue(response, issue):
+    """
+    creates GitHub.Issue EntryContext and results to be printed in Demisto
+    Args:
+        response (dict): The raw HTTP response to be inserted to the 'Contents' field
+        issue (dict or list of dicts): A dictionary or a list of dictionaries formatted for Demisto results
```

Comment:
dict or list

Reply:
you mean in the description for issue in context_create_issue right?